spec/support/admin_view_claim_feature_shared_examples.rb

```diff
@@ -42,7 +42,11 @@
   }
 
   let!(:approved_awaiting_payroll_claim) {
-    eligibility = create(:"#{policy.to_s.underscore}_eligibility", :eligible)
+    eligibility = if policy == Policies::FurtherEducationPayments
+      create(:"#{policy.to_s.underscore}_eligibility", :verified)
+    else
+      create(:"#{policy.to_s.underscore}_eligibility", :eligible)
```

Comment:
Really hate shared example specs 🙃 
I was going to suggest aliasing `eligible` to `verified` in the FE eligibility factory but they kind of have different meanings

Reply:
I think you're right.
The `eligible` trait should also be verified - I'm going to make a separate trait of `not_verified` for the other tests.